src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build_task.py

```diff
@@ -247,3 +247,29 @@ def acr_build_task_logs(cmd,
 
     from .build import acr_build_show_logs
     return acr_build_show_logs(cmd, client, registry_name, build_id, resource_group_name)
+
+
+def acr_build_task_cancel(cmd,
+                          client,
+                          registry_name,
+                          build_id=None,
```

Comment:
I would prefer explicitly canceling a build rather than blindly canceling the last build. The thing is if there are multiple running builds.

Reply:
Removed last build cancellation.